src/components/SelectInput.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FormEventHandler } from 'react'
+
+type Props = {
+    onSubmit: (premises: string) => void
+}
+
+const SelectInput: React.FC<Props> = ({ onSubmit }) => {
+    const handleSubmit: FormEventHandler<HTMLFormElement> = (event) => {
+        event.preventDefault()
+        const premises: string = (event.currentTarget as any)[0].value
+        onSubmit(premises)
+    }
+
+    return (
+        <form onChange={handleSubmit}>
```

Comment:
No need to wrap the select element in a form here.

Reply:
I was getting an error when I used the onChange on the <select> tag. 

<img width="673" alt="Screen Shot 2019-08-13 at 8 43 58 PM" src="https://user-images.githubusercontent.com/34122775/62993151-1bb0cc80-be0b-11e9-8a24-140c5641e350.png">

Reply:
It's a valid prop on the select tag.
What kind of error were you getting?
Looking at the code, i'd guess it's because you copied the handleSubmit function, and the event param is declared as a FormEvent.